modules/user-access/pages/vlac.adoc

```diff
@@ -18,8 +18,8 @@ Sufficient privileges on the base graph or on the global scope are required to p
 * `WRITE_SCHEMA`
  ** Create/run schema change jobs
  ** Create tag-based graphs
-* `READ_DATA, WRITE_DATA`
- ** Run queries that modify the graph
+* `READ_DATA`, `UPDATE_DATA`
```

Comment:
may just change to Data CRUD privileges, since the privileges depends on the query itself

Reply:
```suggestion
* Data CRUD privileges
```